### PR TITLE
fix: updated cleave package, formatted input prefix, chart colors

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15649439
https://tw.auroracreation.com/app/tasks/15655944

Things that I've fixed in this PR:
- Cleave.js package update to make tailPrefix work
- Fixed colors of staking chart progress bar
- Hiding indicator of staking chart when for vote is higher than minimum value

image
image

### CODE REVIEW

File: src/components/v5/common/Fields/InputBase/hooks.ts [hunk 1]
```diff
@@ -83,6 +79,18 @@ export const useFormattedInput = (
     );
   }, []);
 
+  const handlePrefixTrim = useCallback((prefix?: string) => {
+    if (inputRef.current) {
+      const input = inputRef.current;
```
Comment (TomaszChmielPagepro) [inline src/components/v5/common/Fields/InputBase/hooks.ts:84]: I think you can get input element from onChange event

File: src/components/v5/common/Fields/InputBase/FormattedInput.tsx [hunk 2]
```diff
@@ -96,7 +101,10 @@ const FormattedInput: FC<FormattedInputProps> = ({
            */
           options={options}
           onInit={(cleaveInstance) => setCleave(cleaveInstance)}
-          onChange={onChange}
+          onChange={(e) => {
+            handlePrefixTrim(options.prefix);
```
Comment (TomaszChmielPagepro) [inline src/components/v5/common/Fields/InputBase/FormattedInput.tsx:105]: Name of this function is quite misleading as this does not trim the prefix but it's moving the cursor to correct position

**Review by KornelKwiatkowski — DISMISSED** [review]

**Review by krawczykkatarzyna — DISMISSED** [review]

File: src/i18n/en.json [hunk 3]
```diff
@@ -1221,6 +1221,7 @@
     "motion.staking.activateTokens": "Activate tokens",
     "motion.staking.staked": "Staked {value}",
     "motion.staking.input.label": "Stake amount",
+    "motion.staking.input.balance": "Balance {balance}",
```
Comment (rdig) [inline src/i18n/en.json:1224]: Why was this added? 

You have it in the exact next line

File: src/components/v5/common/Fields/InputBase/FormattedInput.tsx (hunk 2 above)
Comment (rdig) [reply src/components/v5/common/Fields/InputBase/FormattedInput.tsx:105]: As I understand it we're no longer doing prefix/suffix token symbol for the staking input. 

@arrenv please confirm if that's still the case

File: src/components/v5/common/ActionSidebar/partials/Motions/consts.ts [hunk 4]
```diff
@@ -26,6 +26,6 @@ export const opposeOption = {
 };
 
 export const STAKING_RADIO_BUTTONS: ButtonRadioButtonItem<MotionVote>[] = [
-  supportOption,
   opposeOption,
+  supportOption,
```
Comment (rdig) [inline src/components/v5/common/ActionSidebar/partials/Motions/consts.ts:30]: This branch will cause a merge conflict with #1394 

Just a heads up

**Review by rdig — CHANGES_REQUESTED** [review]
- "Not public until min 10% is staked" line should only disappear if the motion has that amount of stake, not just when entering a value higher than it

Screenshot from 2023-11-26 02-50-45

Screenshot from 2023-11-26 02-50-49

File: src/components/v5/common/ActionSidebar/partials/Motions/consts.ts (hunk 4 above)
Comment (arrenv) [reply src/components/v5/common/ActionSidebar/partials/Motions/consts.ts:30]: > As I understand it we're no longer doing prefix/suffix token symbol for the staking input.
> 
> @arrenv please confirm if that's still the case

Yes, that is still the case. It was never in the design as far as I understand.

Comment (CzarekDryl) [reply src/components/v5/common/ActionSidebar/partials/Motions/consts.ts:30]: @arrenv @rdig There is token symbol on designs - https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=2729-220432&mode=design&t=wSu6J9exWt1VH7yU-0, or these are old ones?

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Prefix still shows for me in the input when I input a number. It has some confusing behavior when deleting what I input but I guess that will be fixed once we take care of the prefix

**Comment by CzarekDryl** [comment]
I'm closing this PR and moving changes here - https://github.com/JoinColony/colonyCDapp/pull/1427
